### DISCUSSION TITLE
Unnamed-fields format example

### DISCUSSION THREAD

**Discussion by f380cedric** [util-linux/util-linux#1719] [Q&A] [answered]
On line https://github.com/util-linux/util-linux/blob/224410df38d03a572a3b5cd3000906fb1294ce4e/disk-utils/sfdisk.8.adoc#L423,
shouln't this be `*echo -e ',10M,L\n,10M,L\n,+,\n' | sfdisk /dev/sdc*::`? Or even without `+`? BTW, does an empty line work (`*echo -e ',10M,L\n,10M,L\n\n' | sfdisk /dev/sdc*::`)?

**Comment by karelzak** [comment, marked as answer]
Yes, good catch. ",,+" is nonsense, "+" is there to address all available space (size)., so it should be ",+,\n" (as you have in your example).

The empty line is silently ignored, it just repeats a prompt like a normal shell. It's probably the best we can do to make it user-friendly ;-)

**Reply by f380cedric** [reply]
Wouldn't ',,\n' or even ',\n' be enough then? I get that understanding from the following sentence in the manual : " Note '+' is equivalent to the default behaviour for a new partition". Thanks for your time.

**Reply by karelzak** [reply]
Yes,  ',,\n' is supported, but for man-page is probably better to be more verbose :-)
